apps/teslafi/teslafi.go

```diff
@@ -0,0 +1,25 @@
+// Package teslafi provides details for the TeslaFi applet.
+package teslafi
+
+import (
+	_ "embed"
+
+	"tidbyt.dev/community/apps/manifest"
+)
+
+//go:embed teslafi.star
+var source []byte
+
+// New creates a new instance of the TeslaFi applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "teslafi",
+		Name:        "TeslaFi",
+		Author:      "mrrobot245",
+		Summary:     "Shows charge/name/range",
+		Desc:        "Shows your Teslas current Name, Charge in Mi/KM and battery %. Also shows if its charging or not.",
```

Comment:
Will it be obvious to a TeslaFi user how to get the API Key? Would it be helpful to add some pointers in the app description?